blog/content/post/2022-06-17-fobnail_minimal_os.md

```diff
@@ -0,0 +1,253 @@
+| seL4    | No (0) [^1]      | No (0) [^2]      | Yes (+2)          | No (0)            | Limited (-1)   | No (0)                  | Yes (+2)     | Yes (+1)    | Good (+1) [^3]           | Yes (+2)        | OK (but problematic with Genode) (0)  | 7     |
+| Linux   | Yes (+2)         | Yes (+2)         | Yes (+2)          | Yes (+2)          | Yes (+1)       | Yes (kexec) (+2)        | Yes (+2)     | No (0)      | Good (+1) [^5]           | Yes (+2)        |  OK (0)  | 16    |
+| LK      | No (0)           | No (0)           | Limited (-2) [^7] | No (0)            | Yes (+1)       | No (0)                  | Limited (-2) | No (0)      | Good (+1) [^8]           | No (0)          |  OK (0)  | -2    |
+| Fuchsia | Limited (0) [^9] | No (0)           | Yes (+2)          | Limited (0) [^10] | Yes (+1)       | Yes (mexec) (+2)        | Yes (+2)     | Yes (+1)    | Good (+1) [^11]          | No (0)          |  OK (0)  | 7     |
+
+As we can see, multiple OSes were taken into account and a lot of requirements.
+If you are interested in the meanings of the numbers here, please check the
+full [report](https://fobnail.3mdeb.com/minimal-os-for-fobnail/) available on
+Fobnail Project official website.
+
+## Why Linux?
+
+We have decided to use Linux for building minimal OS because it already has
+everything we need, and other OSes would require a significant amount of work.
+In the future, we may use another OS.
+
+## Reference minimal OS for Fobnail Project
+
+We build a minimal OS image for Fobnail Project by using [Yocto
+Project](https://www.yoctoproject.org/) our
+[meta-fobnail](https://github.com/fobnail/meta-fobnail) layer and
+[kas](https://github.com/siemens/kas) container. It is based on
+[TrenchBoot](https://trenchboot.org/) project and releated meta layers:
+[meta-pcengines](https://github.com/3mdeb/meta-pcengines) to provide board
+support package for PCengines apu2 and
+[meta-security](https://git.yoctoproject.org/meta-security/) to use additional
+usefull software like `tpm-tools` package.
+
+### meta-fobnail layer
+
+Our minimal OS is based on the TrenchBoot project, so we use all of their
+components to implement fully secured execution flow: GRUB with SKINIT support,
+Secure Kernel Loader, and Linux with Secure Launch. All of these components we
+got from [TrenchBoot](https://github.com/TrenchBoot/) repositories.
+
+We also add [Fobnail Attester](https://github.com/fobnail/fobnail-attester)
+application with required dependencies: `qcbor` and `libcoap`. The application
+itself is automatically started on system startup. After positive result of
+attestation, we can run the target system from an external device by using
+`kexec`.
+
+The process of generating minimal OS is not complicated. In our case, we were
+using a PC running Ubuntu 20.04 Firstly, we need to download the latest
```

Comment:
```suggestion
using a PC running Ubuntu 20.04. Firstly, we need to download the latest
```

Reply:
fixed https://github.com/3mdeb/news-and-ideas/pull/496/commits/acea6f8ce69e7cd420e09461d14ba223d9597e59#diff-5f2030137d33048b104fbebb6f1f3ce21c3e5522e57c8841c0fc8be5ceb88e2cL122